.github/workflows/ihsBuild.yml

```diff
@@ -328,6 +302,36 @@ jobs:
         with:
           name: sasurl
           path: sas-url.txt
+
```

Comment:
Thanks for fixing this.  Such efforts are necessary to prevent unnecessary cost on our scarce subscription balance.